### PR TITLE
GENERAL / Filters from a widget launches unrequired calculations on itself

### PR DESCRIPTION
**Pull request by Clebal**
Story details: https://app.shortcut.com/cartoteam/story/182594

### CODE REVIEW

File: packages/react-widgets/src/utils/useCustomCompareMemo.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { useRef } from 'react';
+
+export default function useCustomCompareMemo(value, isEqual) {
```
Comment (padawannn) [inline packages/react-widgets/src/utils/useCustomCompareMemo.js:3]: According to the rest of the project, hooks should be in a `hooks` folder so you should rename `utils` by `hooks`

**Review by padawannn — CHANGES_REQUESTED** [review]
I like the new hooks, only a minor request

File: packages/react-widgets/src/utils/useCustomCompareMemo.js (hunk 1 above)
Comment (Clebal) [reply packages/react-widgets/src/utils/useCustomCompareMemo.js]: Done.

**Review by padawannn — APPROVED** [review]

**Comment by VictorVelarde** [comment]
Do you think that it could be useful for custom widgets, maybe?

**Comment by Clebal** [comment]
@VictorVelarde Yes, I thought it would be an useful hook in PS. It's common to create custom widgets.

**Review by VictorVelarde — APPROVED** [review]
Nice work! 🚀

**Pull request merged**